Documentation/advanced-configuration.md

```diff
@@ -31,6 +32,11 @@ To change the prefix of rook Docker images used, the environment variable `ROOK_
 The variable needs to be, depending on the deployment of the rook-operator, added/changed in the rook-operator deployment.
 The default is `rook` which will pull from [Docker Hub](https://hub.docker.com/r/rook/rook/). For example to use your own built images: `your-registry.example.com/your-name/rook`.
 
+## Tweak Ceph mon failover times
+To change the mon failover times, you can add/change the following environment variables for the rook-operator deployment:
+* `ROOK_MON_HEALTHCHECK_INTERVAL` (default `20s`) - The interval to check if every mon is in the quorum.
```

Comment:
It would be good to add the default settings to the rook-operator.yaml with comments there. Then we might not even need this section. It's nice when the setting documentation is in the yaml, then in full documentation we can add more details if needed.